[OIDC] Branded desktop and mobile clients for Azure AD (#806)
owncloud/docs-server: modules/admin_manual/pages/configuration/user/oidc/ms-azure-setup.adoc

    ],
----

=== Register ownCloud Desktop and Mobile Clients

To allow the ownCloud clients (Web/desktop/Android/iOS) to interact with the identity provider, you have to register them as clients.

> Only branded ownCloud desktop and mobile clients can be registered with Azure AD, because they need to be build with customized `scope` and `prompt` parameters. ownCloud full branding subscription is needed. Please get in touch with ownCloud Consulting for more help.

Reviewer: > Please get in touch with ownCloud Consulting for more help

Should it be `support` instead of consulting? This would be the first mentioning of consulting as far I can remember.
In case, I could make a suggestable commit using links we use when mentioning support.

Author: I saw consulting was a used a few times: https://www.google.com/search?q=site%3Adoc.owncloud.com%2Fserver+consulting

@JRundfeldt help with Azure AD + branded clients: Support or Consulting?

Reviewer: Just fyi, we have links (via our homepage) to support but not to consulting. Asking to guide customers where to go...

Reviewer 2: "Support" is more adequate, as they are the central ingresspoint for nearly all customer contacts. "Sales" or "your account manager, if you already have an enterprise subscription" would be even more fitting.

Reviewer: Thanks, will rephrase the sentence accordingly.

> Only branded ownCloud desktop and mobile clients can be registered with Azure AD, because they need to be build with customized `scope` and `prompt` parameters. ownCloud full branding subscription is needed. Please get in touch with ownCloud Consulting for more help.

Reviewer 3: ```suggestion
Only branded ownCloud desktop and mobile clients can be registered with Azure AD, because they need to be built with customized `scope` and `prompt` parameters. ownCloud full branding subscription is needed. Please get in touch with ownCloud Consulting for more help.
```

> Only branded ownCloud desktop and mobile clients can be registered with Azure AD, because they need to be built with customized `scope` and `prompt` parameters. ownCloud full branding subscription is needed. Please get in touch with ownCloud Consulting for more help.

Reviewer: ```suggestion
Only branded ownCloud desktop and mobile clients can be registered with Azure AD, because they need to be built with customized `scope` and `prompt` parameters. As prerequisite, an ownCloud Enterprise subscription including full branding is needed. For more details get in touch with {oc-support-url}[ownCloud Support].
```

Author: Full branding subscription is an additional product, not included in the Enterprise subscription (see https://oc.owncloud.com/rs/038-KRL-592/images/ownCloud-ownBrander_datasheet-EN.pdf)

Reviewer: Thanks for the clarification, will fix

> Only branded ownCloud desktop and mobile clients can be registered with Azure AD, because they need to be built with customized `scope` and `prompt` parameters. As prerequisite, an ownCloud Enterprise subscription including full branding is needed. For more details get in touch with {oc-support-url}[ownCloud Support].

Reviewer: ```suggestion
Only branded ownCloud desktop and mobile clients can be registered with Azure AD, because they need to be built with customized `scope` and `prompt` parameters. As prerequisite, an ownCloud full branding subscription is needed. For more details get in touch with {oc-support-url}[ownCloud Support].
```
@michaelstingl corrected, pls check
